hubspot/client.go

```diff
@@ -120,6 +120,50 @@ func (c *Client) CreateContact(contactInput *ContactInput) (*ContactOutput, Erro
 	return &contactOutput, ErrorResponse{}
 }
 
+// UpdateContact updates a Contact in HubSpot
+func (c *Client) UpdateContact(contactID string, contactInput *ContactInput) (*ContactOutput, ErrorResponse) {
+	log.Printf("INFO: attempting to update HubSpot Contact")
+
+	requestBody, err := json.Marshal(contactInput)
+	if err != nil {
+		log.Printf("ERROR: could not marshal the provided contact body, err: %v", err)
+		return nil, ErrorResponse{Status: "error", Message: "invalid contact input"}
+	}
+	r, err := c.request(
+		fmt.Sprintf("%s/crm/%s/objects/contacts/%s?hapikey=%s", c.APIBaseURL, c.APIVersion, contactID, c.APIKey),
+		http.MethodPatch,
+		requestBody)
```

Comment:
I'd prefer to use a single-use variable here as it will make the function call much easier to read.

```suggestion
	}
	
	apiURL := fmt.Sprintf("%s/crm/%s/objects/contacts/%s?hapikey=%s", c.APIBaseURL, c.APIVersion, contactID, c.APIKey)
	r, err := c.request(apiURL, http.MethodPatch, requestBody)
```